editor/src/components/inspector/sections/style-section/className-subsection/className-subsection.tsx

```diff
@@ -225,7 +225,31 @@ const ClassNameControl = betterReactMemo('ClassNameControl', () => {
   const isMenuEnabled = isSettable && elementPaths.length === 1
   const selectedOptionsLength = selectedOptions?.length ?? 0
   const [isExpanded, setIsExpanded] = React.useState(selectedOptionsLength > 0)
-  const toggleIsExpanded = React.useCallback(() => setIsExpanded((current) => !current), [])
+
+  const expandSection = React.useCallback(() => {
+    setIsExpanded(true)
+    window.setTimeout(() => inputRef.current?.focus(), 0)
```

Comment:
if this is toggled rapidly can this fire at the wrong time? I'm thinking that the `setTimeout` call might need cancelling when contracted.

Reply:
Good point. In theory it shouldn't matter since the input will be destroyed, but I'll add the cancellation anyway.

Reply:
Fixed in https://github.com/concrete-utopia/utopia/pull/1624/commits/e64adb9dfc7c75dc6bd8f8ef29c927095564e30e